docs/user-guide/DPO.rst

```diff
@@ -13,6 +13,8 @@ We support both full-parameter DPO training and LoRA DPO training.
 For full-parameter DPO, there exists an actor and a reference model. Actor is initialized with the reference model and fully trainable, while reference model is frozen and used to calculate logprobs for KL-penalty loss (see `DPO paper <https://arxiv.org/pdf/2305.18290.pdf>`__). 
 For LoRA-based DPO, the actor is initialized by reference model + LoRA weights, where only the LoRA weights are trainable. Therefore, it allows us to switch between actor/reference models by simply enabling/disabling LoRA, and there is no need to store two sets of LLM weights.
 
+Besides the vanilla DPO algorithm, we support other variants of DPO algorithms including Identity preference optimization (IPO) and Reward-aware preference optimization (RPO). The algorithm is identified with the ``dpo.preference_loss`` config variable.
```

Comment:
do we not need to label the RPO dataset with a RM? can we include instructions to do that?

if it's complicated it might make sense to make a seperate doc for it

Reply:
Updated a [commit](https://github.com/NVIDIA/NeMo-Aligner/pull/195/commits/d577b176e2e9f20e546ca1ef044f5679f5ae9bbc) and included more explanation of RPO. However, I didn't include the instruction to generate rewards, which is better covered by the reward model section instead of RPO.